server/migrations/20210209171106_api-key.js

```diff
@@ -0,0 +1,24 @@
+exports.up = async function(knex) {
+    await knex.schema.createTable('user_api_key', table => {
+        table.string('api_key', 64).primary() // sha256 in hex representation is 64 chars
```

Comment:
Should we call this column `sha256_api_key` to make it clearer that it's not the actual API key?

Reply:
Sure, or just hashed_api_key?

Reply:
Even better :+1: